### PR TITLE
Docs/forgotten updates

### PR DESCRIPTION
**Pull request by blankdots**
### Describe the pull request:

- [x] Documentation change

### Pull request long description:

Reflect new changes in mq and db
### Changes made:

1. updated MQ image
2. update MQ queues and message structure information
3. update DB images
4. update db tables
5. add mapper service
6. ignore other files than `.rst` as svg might contain some errors

### Mentions:

not sure i got the mq right @jbygdell might want to check it.

### CODE REVIEW

File: docs/connection.rst [hunk 1]
```diff
@@ -77,17 +86,31 @@ exchange are also internal to CentralEGA.
...
 +=================+=======================================+
+| archived        | Archived files                        |
++-----------------+---------------------------------------+
+| backup          | Signal files to backup                |
++-----------------+---------------------------------------+
+| completed       | Files are backed up                   |
++-----------------+---------------------------------------+
+| error           | User-related errors                   |
++-----------------+---------------------------------------+
 | files           | Trigger for file ingestion            |
```
Comment (jbygdell) [inline docs/connection.rst:103]:
```suggestion
| files           | Receive notification for ingestion from ``CEGAMQ`` |
```

File: docs/connection.rst [hunk 2]
```diff
@@ -222,6 +261,19 @@ corresponding file and send a message to ``completed`` queue.
...
+``Mapper`` service after the file has been published should receive a message
+containing accession IDs mapping between files and datasets
+
+.. code-block:: javascript
+
+   {
+      "user":"john",
+      "filepath":"somedir/encrypted.file.c4gh",
+      "dataset_id": "EGAD001",
+      "accession_ids": ["EGAF0001", "EGAF0002"]
```
Comment (jbygdell) [inline docs/connection.rst:273]: Maybe these should be in the correct format i.e 11 digits.

**Review by pontus — APPROVED** [review]

File: docs/db.rst [hunk 3]
```diff
@@ -243,53 +249,54 @@ check_session_keys_checksums_sha256
 """""""""""""""""""""""""""""""""""
 It returns if the session key checksums are already found in the database.
 
-* Inputs: checksums
+* Inputs: ``checksums``
```
[the remark this answers is not in the document]
Comment (pontus) [reply docs/db.rst:252]: After some additional thinking, I think we should probably get rid of all functions from the DB.

File: docs/connection.rst [hunk 4]
```diff
@@ -77,17 +86,31 @@ exchange are also internal to CentralEGA.
...
 +-----------------+---------------------------------------+
-| archived        | The file is in the archive            |
+| inbox           | Notifications of uploaded files       |
++-----------------+---------------------------------------+
+| ingest          | Trigger for file ingestion            |
 +-----------------+---------------------------------------+
-| stableIDs       | Receive Accession IDs from ``CEGAMQ`` |
+| mappings        | Received Dataset to file mapping      |
++-----------------+---------------------------------------+
+| accessionIDs    | Receive Accession IDs from ``CEGAMQ`` |
```
[the remark this answers is not in the document]
Comment (blankdots) [reply docs/connection.rst:112]: done

File: docs/db.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (blankdots) [reply docs/db.rst:252]: until we change them we can keep on documenting them. I would say

[the remark this answers is not in the document]
Comment (blankdots) [reply docs/db.rst:252]: @pontus that is ok from my point

**Review by jbygdell — APPROVED** [review]

File: docs/db.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (blankdots) [reply docs/db.rst:252]: but let us keep the changes here until we merge the DB changes

**Pull request merged**